src/phoenix/server/api/types/Project.py

```diff
@@ -0,0 +1,72 @@
+from datetime import datetime
+from typing import Optional
+
+import strawberry
+from strawberry.types import Info
+
+from phoenix.server.api.context import Context
+
+from .node import Node
+
+
+@strawberry.type
+class Project(Node):
+    name: str
+
+    @strawberry.field
+    def start_time(
+        self,
+        info: Info[Context, None],
+    ) -> Optional[datetime]:
+        if (traces := info.context.traces) is None:
+            return None
+        start_time, _ = traces.right_open_time_range
+        return start_time
+
+    @strawberry.field
+    def end_time(
+        self,
+        info: Info[Context, None],
+    ) -> Optional[datetime]:
+        if (traces := info.context.traces) is None:
+            return None
+        _, end_time = traces.right_open_time_range
+        return end_time
```

Comment:
technically this is the `stop_time`. to get the `end_time` it needs to subtract a small amount

if stop_time = 23:15:00.000, then end_time = 23:14:99.999